R/object_name_linter.R

```diff
@@ -143,17 +180,17 @@ check_style <- function(nms, style, generics = character()) {
       # If they are not conforming, but are S3 methods then ignore them
       conforming[!conforming][has_generic] <- TRUE
     }
-    # exclude namespace hooks like .onLoad, .Last.lib, etc (#500)
-    is_special <- is_special_function(nms[!conforming])
+    # exclude namespace hooks like .onLoad, .Last.lib, etc (#500) and ...
+    is_special <- is_special_function(nms[!conforming]) | nms[!conforming] == "..."
```

Comment:
nice catch! is this covered by the `function(...) ..1 + ..2` test below?

PS I'm surprised `..1`/`..2` are excepted then?

Reply:
`..1` and `..2` aren't checked, since they are only used, but neither assigned nor explicitly declared as formals.

Reply:
And yes, I discovered the "bug" when writing the test you mention.

Reply:
oh right, but `...` are "assigned" as `SYMBOL_SUB`. thanks!